ansible/roles/upgrade/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+- include_tasks: looped_tasks.yml
+  with_items:
+    - 2.3.5
+    - 2.4.9
+    - 2.5.7
+    - 2.6.4
+    - 2.7.2
+
+  loop_control:
+    loop_var: version
+
+- name: install redis server
+  become: true
+  apt:
+    name: redis-server
+    state: present
+    update_cache: yes
+
+- name: update board_report dataset
+  become: true
+  become_user: postgres
+  command: psql ckan_default -c "UPDATE resource SET format = 'CSV' WHERE package_id = '2c5c458b-8a2f-4319-bd6a-58fd00b70e74';"
```

Comment:
This scares me, can we have a comment?

What's the arbitrary string?

Reply:
Fixed in 8810e60